.storybook/theme.js

```diff
@@ -0,0 +1,22 @@
+import {create} from '@storybook/theming'
+
+export default create({
+    base: 'light',
+    brandTitle: 'Libero Publisher',
+    brandUrl: 'https://libero.pub/',
+    colorPrimary: 'rgb(2, 136, 209)',
```

Comment:
Done an experiment locally, and reasonably straightforward to get this values from Sass using Webpack.